docs/sources/release-notes/v2.3.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Grafana Mimir version 2.3 release notes"
+menuTitle: "V2.3 release notes"
+description: "Release notes for Grafana Mimir version 2.3"
+weight: 300
+---
+
+# Grafana Mimir version 2.3 release notes
+
+Grafana Labs is excited to announce version 2.3 of Grafana Mimir, the most scalable, most performant open source time series database in the world.
+
+The highlights that follow include the top features, enhancements, and bugfixes in this release. If you are upgrading from Grafana Mimir 2.2, there is upgrade-related information as well.
+For the complete list of changes, see the [Changelog](https://github.com/grafana/mimir/blob/main/CHANGELOG.md).
+
+## Features and enhancements
+
+- **Support for OpenTelemetry Metrics ingestion path**:
+  Grafana Mimir now has experimental support for ingesting metrics from the [OpenTelemetry Collector](https://opentelemetry.io/docs/collector/) via two options: `prometheusremotewrite` and `otlphttp`. There is also experimental support for the OpenTelemetry Metrics ingestion path on `/otlp/v1/metrics`.
```

Comment:
Remote write was already supported. In this release we're introducing support for `otlphttp`.

```suggestion
  In addition to the [OpenTelemetry Collector](https://opentelemetry.io/docs/collector/) exporter `prometheusremotewrite`, in this release Grafana Mimir introduces an experimental support for ingesting metrics using the `otlphttp` exporter. The `otlphttp` API endpoint exposed by Mimir is at the path `/otlp/v1/metrics`.
```